dev/vagrant-config/scripts/php.sh

```diff
@@ -7,7 +7,7 @@ echo " "
 debconf-set-selections <<< 'libssl1.0.0:amd64 libssl1.0.0/restart-services string ntp'
 
 # removing php-pear php-dev as they rely on PHP5
-apt-get install -y haveged php7.0-fpm php7.0-mysql php7.0-apcu php7.0-json php7.0-curl php-mbstring php7.0-xml php7.0-zip php7.0-xdebug > /dev/null 2>&1
+apt-get install -y haveged php7.0-fpm php7.0-mysql php7.0-apcu php7.0-json php7.0-curl php-mbstring php7.0-xml php7.0-zip php7.0-xdebug php7.0-sqlite3 sqlite > /dev/null 2>&1
```

Comment:
ah we don't, i was going to use it for the test in vagrant but then doctrine migrations complain so created the hms_test DB instead